@3rdweb-sdk/react/useContractBadge.ts

```diff
@@ -0,0 +1,14 @@
+import { useQuery } from "@tanstack/react-query";
+
+export function useContractBadge(address: string) {
+  return useQuery(["badge", "contract", address], async () => {
+    const res = await fetch(`/api/badges/contract`, {
+      method: "POST",
+      body: JSON.stringify({ address }),
+    });
+    if (res.status >= 400) {
+      throw new Error(await res.json().then((r) => r.error));
+    }
+    return res.json() as Promise<any>;
+  });
+}
```

Comment:
ideally we don't need this and instead we can just make a normal GET request to the api route with query param of chainId + contract address -> it returns the svg as an image (so then u can put it in an <img src="" /> for example)